spikeinterface/sorters/runsorter.py

```diff
@@ -197,6 +197,8 @@ def append_parent_folder(p):
 
 def path_to_unix(path):
```

Comment:
maybe this should then rather be called `customize_path_for_os`?

Reply:
I like the ideia
What do you think? @alejoe91

Reply:
Fine for me, but we'd always use it in 'unix' mode no?

Reply:
:+1: Yes, if that's the only system you use in the containers then we can also leave it as it is.